RELEASES.md

```diff
@@ -1,79 +1,140 @@
-Version 1.0.0-beta (April 2015)
--------------------------------------
-
-* ~1100 changes, numerous bugfixes
```

Comment:
@brson I see you put the count back in 94caeb326c45bd0a108abe92ae50f9deda0aeb4c... kool.